### DISCUSSION TITLE
ring-mqtt for RTSP without MQTT

### DISCUSSION THREAD

**Discussion by gapthorpe** [tsightler/ring-mqtt#905] [Support Q&A] [answered]
Hi
Have just discovered this project while looking into a way to support getting an RTSP feed from my Ring cameras.  Appreciate that the project's main focus is on the MQTT function, but is there a way to run which just surfaces the RTSP feed without MQTT?  If not, could that be considered as a potential future feature?

Thanks!

**Comment by tsightler** [comment, marked as answer]
Just set the MQTT URL to `mqtt://127.0.0.1:51883` and it will effectively connect to itself.  Since this broker does not listen outside of the container it's safe.

**Reply by gapthorpe** [reply]
Perfect, thank you!
